### ISSUE TITLE
[FEATURE]: Add Waves for NavBar

### ISSUE THREAD

**Issue by AbhijithGanesh** [ComputerSocietyVITC/Annual-Yearbook#49]
### Description

According to [Design](https://github.com/ComputerSocietyVITC/Annual-Yearbook/blob/master/Design/annual-yearbook_landing.png?raw=true), We need the Navbar with waves. 

### Additional information

Linked Issue: #46

**Comment by Shansince2k3** [comment]
Hi @AbhijithGanesh  I would like to work on this. Could you please assign this to me ?

**Comment by Ayusht777** [comment]
your website link is not working

**Comment by SumanJain2005** [comment]
Hey @AbhijithGanesh, can you please assign me the issue?

**Comment by AbhijithGanesh** [comment]
Heyy Suman, I am not sure if this is a good issue

**Comment by SumanJain2005** [comment]
yeah, but i though about giving it a try, even though the desgin for the navbar seemed challenging :) , anyways thanks for replying
